packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -7,7 +7,7 @@
 sudo apt-get install -yqq nginx
 git clone https://github.com/h5bp/server-configs-nginx.git
 cd ./server-configs-nginx
-git checkout tags/2.0.0
+git checkout tags/5.0.1
```

Comment:
According to git blame this line was added by @edmofro in late 2021. I'm curious if you have any memory why we picked `2.0.0`, that release was already 4 years old at that point. Is there a specific reason we want to use `2.0.0`? If so we'd need to undo these uses of `compression.conf`

Reply:
Some informed speculation on my reasoning: it's what we were using at the time, and to limit the scope of what was quite a big undertaking (moving us to blue/green deployments using a base AMI), I kept it as is so that I didn't have to review breaking changes.